### PR TITLE
Update Stanford DC with new tables; Fix local mixer api root issue

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: deploy/overlays/stanford/custom_bigtable_info.yaml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 project: datcom-stanford
 instance: dc-graph
 tables:
-  - private_2022_11_09_19_28_52
+  - smokepm_2023_03_01_15_05_49
+  - aqicn_2023_02_24_11_39_31
```
Comment (Fructokinase) [inline deploy/overlays/stanford/custom_bigtable_info.yaml:5]: This can be be updated to aqicn_2023_03_01_19_03_31

**Review by Fructokinase — APPROVED** [review]
Thanks! 

nit: would it help to have an extra option in run_server.sh that also sets USE_LOCAL_MIXER?

File: deploy/overlays/stanford/custom_bigtable_info.yaml (hunk 1 above)
Comment (shifucun) [reply deploy/overlays/stanford/custom_bigtable_info.yaml]: Done

**Comment by shifucun** [comment]
> Thanks!
> 
> nit: would it help to have an extra option in run_server.sh that also sets USE_LOCAL_MIXER?

Right, now there is "-l" to set this var

**Pull request merged**